src/rrdp-iostat/rrdp_iostat.ml

```diff
@@ -670,7 +670,7 @@ module Iostats_value = struct
       owner, ds_make ~name:(key_format "latency")
         ~description:"Average I/O latency"
         ~value:(Rrd.VT_Float value.latency)
-        ~ty:Rrd.Gauge ~units:"milliseconds" ~min:0. ();
+        ~ty:Rrd.Gauge ~units:"μs" ~min:0. ();
```

Comment:
This uses an UTF-8 character and we don't pay attention to UTF-8 when we work with strings. This will probably survive it to where you need this but it would be safer to stick with the ASCII subset.

Reply:
Note there are already more instances of utf-8 in this code.